.github/workflows/style_check.yml

```diff
@@ -0,0 +1,47 @@
+name: Style check
```

Comment:
This can be removed, it is redundant with the pre-commit configuration. The one piece that might be handy to copy would be the "single requirement for branch merge" job at the end.